lkp-exec/install

```diff
@@ -225,11 +225,15 @@ install_packages() {
 			install_ocfs2_tools "$ocfs2_tools_name"
 		fi
 	fi
+	
+	if [ "$distro" = "exaleap-riscv-linux" ]; then
+		echo "riscv64"
+	fi
 
 	$LKP_SRC/distro/installer/$distro $generic_packages || {
 		echo "Cannot install some packages in $LKP_SRC/distro/depends/${script}"
 		exit 1
-	}
```

Comment:
this change is not necessary

Reply:
> this change is not necessary

Removed it.

Reply:
seems this change of line 236 still exists, need further check